examples/gallery/lines/decorated_lines.py

```diff
@@ -0,0 +1,84 @@
+"""
+Decorated lines
+---------------
+To draw a so-called *decorated line*, i.e., symbols along a line
+or curve, use the ``style`` parameter of the
+:meth:`pygmt.Figure.plot` method with the argument ``"~"`` and the
+desired modifiers. A colon (``":"``) is used to separate the
+algorithm settings from the symbol information.
+ This example shows how to adjust the symbols.
+Beside the build-in symbols also custom symbols can be used.
+For modifying the base line via the ``pen`` parameter, see the
```

Comment:
```suggestion
For modifying the baseline via the ``pen`` parameter, see the
```

Reply:
All occurencies are updated in commit https://github.com/GenericMappingTools/pygmt/pull/2564/commits/d0d7c59463bf72d8f5736abfaadca701cec68825.